app/facets/participant/game-master.coffee

```diff
@@ -0,0 +1,22 @@
+{nx} = require 'nexus-node'
+
+class GameMaster
+
+	constructor: (@participant) ->
+
+		@participant.game_master = @
+
+		@puzzles = new nx.Cell
+			'<<-*': [@participant.game_session, 'puzzles']
```

Comment:
coming soon to nexus wiki: everything you didn't want to know about bindings :cat: